libs/features/personalization/personalization.js

```diff
@@ -11,6 +11,47 @@ const ENT_CACHE_EXPIRE = 1000 * 60 * 60 * 3; // 3 hours
 const ENT_CACHE_REFRESH = 1000 * 60 * 3; // 3 minutes
 const PAGE_URL = new URL(window.location.href);
 
+export const appendJsonExt = (path) => (path.endsWith('.json') ? path : `${path}.json`);
+
+export const normalizePath = (p) => {
+  let path = p;
+
+  if (!path.includes('/')) {
+    return path;
+  }
```

Comment:
You could save a couple of bits and just do `if (!path.includes('/')) return path;`

Reply:
agreed. updated.